react/src/components/UserResourcePolicySettingModal.tsx

```diff
@@ -71,105 +92,202 @@ const UserResourcePolicySettingModal: React.FC<Props> = ({
-        //     name: userResourcePolicy || "",
-        //     props: {
-        //       max_vfolder_count: values?.max_vfolder_count,
-        //       max_quota_scope_size: GBToBytes(values?.max_vfolmax_quota_scope_sizeder_count),
-        //     },
-        //   },
-        //   onCompleted(response) {
-        //     if (response?.create_user_resource_policy?.ok) {
-        //       message.success(
-        //         t("storageHost.ResourcePolicySuccessfullyCreated")
-        //       );
-        //     } else {
-        //       message.error(response?.create_user_resource_policy?.msg);
-        //     }
-        //     onRequestClose();
-        //   },
-        //   onError(error) {
-        //     console.log(error);
-        //     message.error(error?.message);
-        //   },
-        // });
-      }
-    });
+            onCompleted(res, errors) {
+              if (!res?.create_user_resource_policy?.ok || errors) {
+                message.error(res?.create_user_resource_policy?.msg);
+                onRequestClose();
+              } else {
+                message.success(
+                  t('storageHost.ResourcePolicySuccessfullyUpdated'),
+                );
+                onRequestClose(true);
+              }
+            },
+            onError(error) {
+              message.error(error?.message);
+            },
+          });
+        } else {
+          commitModifyUserResourcePolicy({
+            variables: {
+              name: values?.name,
+              props: props as ModifyUserResourcePolicyInput,
+            },
+            onCompleted(res, errors) {
+              if (!res?.modify_user_resource_policy?.ok || errors) {
+                message.error(res?.modify_user_resource_policy?.msg);
+                onRequestClose();
+              } else {
+                message.success(
+                  t('storageHost.ResourcePolicySuccessfullyUpdated'),
+                );
+                onRequestClose(true);
+              }
+            },
+            onError(error) {
+              message.error(error?.message);
+            },
+          });
+        }
+      })
+      .catch(() => {});
   };
 
   return (
     <BAIModal
-      {...baiModalProps}
+      title={
+        userResourcePolicy === null
+          ? t('resourcePolicy.CreateResourcePolicy')
+          : t('resourcePolicy.UpdateResourcePolicy')
+      }
+      onOk={handleOk}
+      onCancel={() => onRequestClose()}
       destroyOnClose
-      title={t('storageHost.ResourcePolicySettings')}
-      onOk={_onOk}
+      confirmLoading={
+        isInFlightCommitCreateUserResourcePolicy ||
+        isInFlightCommitModifyUserResourcePolicy
+      }
+      {...baiModalProps}
     >
       <Alert
         message={t('storageHost.BeCarefulToSetUserResourcePolicy')}
         type="warning"
         showIcon
-        style={{ marginTop: 20, marginBottom: 25 }}
+        style={{ marginTop: token.marginMD, marginBottom: token.marginMD }}
```

Comment:
How about removing `marginTop`?
```suggestion
        style={{ marginBottom: token.marginMD }}
```